browser/help/topics/faf.md

```diff
@@ -9,10 +9,4 @@ Note that this annotation contains filtering allele frequency information from t
 
```

Comment:
"Note that this annotation contains filtering allele frequency information from the genetic ancestry group with the highest **FAF**" 

We should clarify that this part is referring to the grpmax faf (or whatever we settled on for a name) and not just faf

Reply:
will add a sentence -- let me know what you think